>From df55066f460a3b2da7ee05d013c65b8f6342fd78 Mon Sep 17 00:00:00 2001
From: Carlos Maddela <maddela@labyrinth.net.au>
Date: Sat, 1 Feb 2014 09:14:40 +1100
Subject: fix delete link

Description: Fix "Delete all listed files" link for unreferenced
 files.
Patch-Name: fix-delete-link.patch
---
 source/expiration.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/expiration.cc b/source/expiration.cc
index 65079fb..40bf4ee 100644
--- a/source/expiration.cc
+++ b/source/expiration.cc
@@ -475,7 +475,7 @@ void expiration::Action(const string & cmd)
 
 		mstring delURL(cmd);
 		StrSubst(delURL, "justShow", "justRemove");
-		SendFmtRemote << "<a href=\"/"<<delURL<<"\">Delete all listed files</a> "
+		SendFmtRemote << "<a href=\""<<delURL<<"\">Delete all listed files</a> "
 				"(no further confirmation)<br>\n";
 		return;
 	}
